backend/cic/settings/settings.py

```diff
@@ -0,0 +1,36 @@
+from .base import *  # noqa: F401,F403
+from opensearchpy.connection import RequestsHttpConnection
+from opensearchpy import OpenSearch
+
+ENVIRONMENT = os.getenv('CIC_ENVIRONMENT')
+
+if ENVIRONMENT == 'dev':
+    from .dev import * 
+elif ENVIRONMENT == 'prod':
+    from .prod import *  # noqa: F403
+    
+AWS_S3_BUCKET_AUTH_STATIC = True
+STATICFILES_STORAGE = "django_s3_storage.storage.StaticS3Storage"
+
+AWS_S3_CUSTOM_DOMAIN = '%s.s3.amazonaws.com' % AWS_S3_BUCKET_NAME_STATIC  # noqa: F405
+SECURE_REFERRER_POLICY = 'strict-origin-when-cross-origin'
+STATIC_URL = "https://%s.s3.amazonaws.com/" % AWS_S3_BUCKET_NAME_STATIC
+
+OPENSEARCH_URL = os.getenv('OPENSEARCH_URL', 'vpc-cicecluster-hk6ksspjhvgpkedai3sp5m6a2a.us-east-1.es.amazonaws.com')
```

Comment:
You shouldn't have CU-specific AWS account info in this public repo. These should be externalized to environment variables.

Reply:
Done. It's already in an env variable called "OPENSEARCH_URL". I just forgot to remove it from the settings. My bad.
Thanks for reviewing.